core/src/main/java/jenkins/model/lazy/AbstractLazyLoadRunMap.java

```diff
@@ -443,7 +455,8 @@ private Index copy() {
      * 
      * @return null if the data failed to load.
      */
-    protected R load(int n, Index editInPlace) {
+    private R load(int n, Index editInPlace) {
```

Comment:
This breaks API compatibility.  Doesn't seem to be anyone using this in the jenkinsci github org though